app/src/main/java/uc/edu/klopfsea/groupscheduler/model/EventList.kt

```diff
@@ -4,7 +4,7 @@ object EventList {
     val eventList = arrayListOf<Event>()
 
     fun getData(): ArrayList<Event> {
-        eventList.add(Event(isDaysSet = true, isDateSet = false, date = "", days = "M,T,W,R", time = "", timeSpecification = "All Day", period = ""))
+        eventList.add(Event(isDaysSet = true, isDateSet = false, date = "", days = "M,T,W,R,SA,SUN", time = "", timeSpecification = "All Day", period = ""))
         eventList.add(Event(isDaysSet = false, isDateSet = true, date = "12th", days = "", time = "4:30", timeSpecification = "Before 4:30 PM", period = "PM"))
         return eventList
     }
```

Comment:
I added friday, saturday, and sunday. unless you guys were only doing certain days.